src/browser/documentation/guides/sandboxMovies.tsx

```diff
@@ -0,0 +1,1120 @@
+import React from 'react'
```

Comment:
Is this file used for anything?

Reply:
It's not! Should have a done a quick self review and removed it 😓